cmd/manager/main.go

```diff
@@ -7,6 +7,7 @@ import (
 	"os"
 	"runtime"
 
+	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.
```

Comment:
This line is not required. Next line content is the same.

Reply:
I think it would not hurt if we leave, since that's a part of every Kubernetes client. So if we have developers working against those clouds, our project is already compatible, maybe we have more reasons to let it stay than to remove it.

Reply:
I was not referring to the import statement. The comment above that is (line 10 and 11) are the same content.

Reply:
Check! I missed that one, but it's gone now.